middle_end/flambda/from_lambda/closure_conversion.ml

```diff
@@ -198,6 +199,7 @@ let tupled_function_call_stub
       ~recursive
   in
   func_decl, code_id, params_and_body
+*)
```

Comment:
Did you forget to remove that rather than comment it ?

Reply:
It has now been removed, ^^